src/test/java/seedu/planner/TestApp.java

```diff
@@ -41,12 +42,10 @@ public TestApp(Supplier<ReadOnlyModulePlanner> initialDataSupplier, Path saveFil
         this.saveFileLocation = saveFileLocation;
 
         // If some initial local data has been provided, write those to the file
-        /*
         if (initialDataSupplier.get() != null) {
-            createDataFileWithData(new XmlSerializableAddressBook(this.initialDataSupplier.get()),
+            createDataFileWithData(new JsonSerializableModulePlanner(this.initialDataSupplier.get()),
```

Comment:
Can you revert change for this file? I am working on my storage tests and I will touch this file, so want to avoid merge conflict.

Reply:
Okay sure